guide/additional-info/changes-in-v12.md

```diff
@@ -14,6 +14,34 @@ For now, you do need Git installed and added to your PATH environment, so ensure
 * The use of parenthesis designates optional inclusion. For example, `Channel#fetch(Pinned)Message(s)` means that this section will include changes for `Channel#fetchPinnedMessages`, `Channel#fetchMessages`, and `Channel#fetchMessage`.
 * The use of asterisks designates a wildcard. For example, `Channel#send***` means that this section will include changes for `Channel#sendMessage`, `Channel#sendFile`, `Channel#sendEmbed`, and so forth.
 
+### Collection
+
+#### Collection#exists
+
+`collection.exists` was removed entirely, `collection.some()` should be used to check if an element exists in the collection that satisfies the provided value.
+
+```diff
+- client.users.exists('username', 'Bob');
++ client.users.some(user => user.username === 'Bob');
+```
+
+#### Collection#filterArray
+
+`colelction.filterArray()` was removed entirely, as it was just a helper method for `collection.filter().array()` and most of the time converting a collection to an array is an unnecessary step.
```

Comment:
Typo, `colelction` should be `collection`

Reply:
Fixed.